deployment/instance/instance.go

```diff
@@ -100,6 +100,13 @@ func (i *instance) WaitUntilReady(
 			sshTunnel := i.sshTunnelFactory.NewSSHTunnel(sshTunnelOptions)
 			go sshTunnel.Start(sshReadyErrCh, sshErrCh)
 
+			go func() {
+				for {
```

Comment:
Currently this for loop will never terminate, and will hard loop once the `sshErrCh` is closed.

This can be improved by using:
```
for err := range sshErCh {
  ...
}
```